cmd/boulder-wfe2/main.go

```diff
@@ -308,12 +297,8 @@ func main() {
 	if c.WFE.NonceHMACKey.KeyFile != "" {
```

Comment:
I don't think we need this check anymore, the validator tags should be doing this for us:

```go
NonceHMACKey cmd.HMACKeyConfig `validate:"required"`
```

If we want to keep this around we should invert it and then `cmd.Fail()` if it's not specified. This allows us to remove the indentation on lines 298-299.